addons/l10n_pe_website_sale/demo/website_demo.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="1">
+
+    <record id="default_website_pe" model="website">
+        <field name="name">(PE) Peruvian Website</field>
+        <field name="company_id" ref="l10n_pe.demo_company_pe" />
+        <field name="logo" type="base64" file="l10n_pe_website_sale/static/description/icon.png" />
+        <field
+            name="domain"
+            model="ir.config_parameter"
+            eval="obj().env['ir.config_parameter'].sudo().get_param('web.base.url')"
+        />
```

Comment:
I really finds this way of formatting odd, would you mind writing it like this instead:
```suggestion
        <field
            name="domain"
            model="ir.config_parameter"
            eval="obj().env['ir.config_parameter'].sudo().get_param('web.base.url')"/>
```
IMO it lightens the file and is easier to read since there are less lines